src/core/api/track_manager.ts

```diff
@@ -574,13 +579,19 @@ export default class TrackManager {
                                                       null;
 
     return audioInfos.adaptations
-      .map((adaptation) => ({
-        language: takeFirstSet<string>(adaptation.language, ""),
-        normalized: takeFirstSet<string>(adaptation.normalizedLanguage, ""),
-        audioDescription: adaptation.isAudioDescription === true,
-        id: adaptation.id,
-        active: currentId == null ? false : currentId === adaptation.id,
-      }));
+      .map((adaptation) => {
+        const formatted : ITMAudioTrackListItem = {
+          language: takeFirstSet<string>(adaptation.language, ""),
+          normalized: takeFirstSet<string>(adaptation.normalizedLanguage, ""),
+          audioDescription: adaptation.isAudioDescription === true,
+          id: adaptation.id,
+          active: currentId == null ? false : currentId === adaptation.id,
+        };
+        if (adaptation.isDub === true) {
```

Comment:
So `dub` can be only `true` or `undefined` ? No `false` value possible ?

Reply:
Yep thought about that.

The problem is that we know that an audio track is a dub through the presence of a Role element in the MPD specifying it is one.

The absence of that element does not obligatory means that it is not a dub.
We could also define `isDub` to be false when the Role is explicitely `main` but I was afraid of packagers not respecting that spec.